### PR TITLE
Updated boss-account.py to create alarms

### PR DESCRIPTION
**Pull request by sandyhider**
- Updated bin/boss-account.py so it correctly creates billing alarms now.
- It was trying using the sns client instead of cloudwatch to create the alarms.
- Updated several locations in the code changing BILLING_THREASHOLDS to BILLING_THRESHOLDS.

### CODE REVIEW

File: bin/boss-account.py [hunk 1]
```diff
@@ -158,6 +159,9 @@ def __init__(self, bosslet_config):
         list = AlertList(args.bosslet_config)
 
     if args.create:
+        if list.create() is False:
```
Comment (derek-pryor) [inline bin/boss-account.py]: Why did you add this? if the list already exists it will always try to create it. These two lines mirror lines 168/169.

File: bin/boss-account.py [hunk 2]
```diff
@@ -108,18 +109,18 @@ def create(self):
             'Namespace': 'AWS/Billing',
             'Statistic': 'Maximum',
             'Dimensions': [{'Name': 'Currency', 'Value': currency}],
-            'Period': 10,
+            'Period': 21600,
```
Comment (derek-pryor) [inline bin/boss-account.py]: Why change this, what is it doing? (I just copied the original code so I'm not sure what it is setting)

File: bin/boss-account.py [hunk 3]
```diff
@@ -34,6 +34,7 @@ class SubscriptionList(object):
     def __init__(self, bosslet_config, topic):
         self.bosslet_config = bosslet_config
         self.client = bosslet_config.session.client('sns')
+        self.client_cw = bosslet_config.session.client('cloudwatch')
```
Comment (derek-pryor) [inline bin/boss-account.py]: Add this to the `BillingList` class, as it is specific to that logic

File: bin/boss-account.py (hunk 2 above)
Comment (sandyhider) [reply bin/boss-account.py]: First off it fails with 10 now.  I think we tried at one time to change it to get better resolution then 6 hours.  changing it to a lower number doesn't actually change the how often they can fire. what ends up happening is they all reset if it is less then 6 hours (21600).  So we end  up getting all the emails again.

File: bin/boss-account.py (hunk 1 above)
Comment (sandyhider) [reply bin/boss-account.py]: I'll remove it.  I got into a catch 22 it created the topic but failed to create the alarms, then it would just not try and create them again.  It get it to run again I was changing a few lines.  I thought I had them all back but I guess I added this one by mistake.

Comment (derek-pryor) [reply bin/boss-account.py]: Got ya. I think what we need to do is to create the override `BillingList.exists()` so that it also checks to see if the alarms were created, and if not have it return false. Then in `BillingList.create()` have logic so that it doesn't try to create the SNS topic is it already exists.

File: bin/boss-account.py (hunk 2 above)
Comment (derek-pryor) [reply bin/boss-account.py]: Maybe a comment that this value cannot be less than 6hrs (or something)
Should this be an optional argument?
